docs/modules/ROOT/pages/index.adoc

```diff
@@ -444,7 +464,100 @@ The signature type is represented by the `sig_type` field and must be one of the
+
+[mermaid,width=100%]
+....
+erDiagram
+  CH["Credential holder"] {}
+  CHVC["Credential holder’s verifiable credential"] {}
+  TBS["`tbs` data structure"] {}
+  IVC["Identity assertion’s verifiable credential"] {}
+  ASSET["C2PA asset"] {}
+
+  CH ||--|| CHVC : "controls via private key"
+  IVC||--|| TBS: "references"
+  IVC ||--|| CHVC: "issued by"
+  TBS ||--|| ASSET: "describes"
+....
+
+==== Credential holder’s verifiable credential
+
+TO DO: Specify requirements for CH VC.
+
+* Needs to have discoverable `assertionMethod`
+* DID method recommendations/requirements?
+* Alignment with ToIP verifiable identifier
+
+==== Identity assertion’s verifiable credential
+
+The _<<_c2pa_claim_generator,C2PA claim generator>>_ MUST generate the `tbs` data structure as described in *TO DO: Xref tbs once merged* and present that to the _<<_credential_holder,credential holder>>_ for signature.
+
+The _<<_credential_holder,credential holder>>_ MUST issue a _new_ _<<_w3c_verifiable_credential,W3C verifiable credential>>_ which satisfies the following requirements:
+
+* The credential MUST meet all requirements for a verifiable credential as described in https://www.w3.org/TR/vc-data-model-2.0/[Verifiable Credentials Data Model, version 2.0].
+* The `@context` field MUST contain at least the following entries:
+** `"https://www.w3.org/ns/credentials/v2"`
+** `"https://creator-assertions.github.io/tbd/tbd"`
+* The `type` field MUST contain at least the following entries:
+** `"VerifiableCredential"`
+** `"CreatorIdentityAssertion"`
+* The `credentialSubject` field MUST contain a `tbs` entry, which MUST be the JSON serialization of the `tbs` data structure presented for signature. All bytestring entries in the `tbs` data structure (i.e. `hash` entries in hashlink URL data structures) MUST be converted to the corresponding Base-64 plain-text strings. (TO DO: Specify Base-64 encoding options.)
+* The `credentialSubject` field MAY contain other entries. The meaning of any such entries is not defined in this specification.
+* The `issuer` field MUST exist.
+* A suitable `proof` field MUST exist and prove issuance of this credential by `issuer`. (TO DO: Review https://www.w3.org/TR/vc-data-integrity/.)
+* TO DO: Describe how to prove time of issuance. Perhaps an independent field in identity assertion with RFC 3161 timestamp?
+
+The value of the `signature` field in this case is the JSON-LD serialization of the _<<_w3c_verifiable_credential,W3C verifiable credential>>_ as just described.
+
+An example of the identity assertion verifiable credential is given below:
+
+.Example identity assertion verifiable credential
+[#example-ia-vc]
+[source,json]
+----
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://creator-assertions.github.io/tbd/tbd"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "CreatorIdentityAssertion"
+  ],
+  "issuer": "did:example:2g55q912ec3476eba2l9812ecbfe",
+  "credentialSubject": {
```

Comment:
I think that credentialSubject should also have a type field. This type should be defined in the @context and be used for validation purposes.